code/Loom/Catalog/Plugin/Model/Config.php

```diff
@@ -0,0 +1,29 @@
+<?php
+namespace Loom\Catalog\Plugin\Model;
```

Comment:
Add package and copyright:
/*
 * @package    Loom_Catalog
 * @copyright  Loom Oy - 2020
 */